Library/Homebrew/uninstall.rb

```diff
@@ -68,6 +68,10 @@ def self.uninstall_kegs(kegs_by_rack, casks: [], force: false, ignore_dependenci
 
               unversioned_name = f.name.gsub(/@.+$/, "")
               maybe_paths = Dir.glob("#{f.etc}/*#{unversioned_name}*")
```

Comment:
Maybe this glob should be more specific instead e.g. 
```suggestion
              maybe_paths = Dir.glob("#{f.etc}/#{unversioned_name}*")
```